client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -77,56 +66,29 @@ const DesignSelector: React.FunctionComponent = () => {
 			</div>
 			<div className="design-selector__design-grid">
 				<div className="design-selector__grid">
-					{ designs.featured.map( design => {
-						const isFocused = hoverDesign === design.slug || focusDesign === design.slug;
-						return (
-							<Spring
-								native
-								key={ design.slug }
-								from={ ZOOM_OFF }
-								to={ isFocused ? ZOOM_ON : ZOOM_OFF }
-							>
-								{ ( props: React.CSSProperties ) => (
-									<animated.button
-										style={ props }
-										onMouseEnter={ () => setHoverDesign( design.slug ) }
-										onMouseLeave={ () =>
-											setHoverDesign( s => ( s === design.slug ? undefined : s ) )
-										}
-										onFocus={ () => setFocusDesign( design.slug ) }
-										onBlur={ () => setFocusDesign( s => ( s === design.slug ? undefined : s ) ) }
-										className="design-selector__design-option"
-										onClick={ () => {
-											setSelectedDesign( design );
+					{ designs.featured.map( design => (
+						<button
+							key={ design.slug }
+							className="design-selector__design-option"
+							onClick={ () => {
+								setSelectedDesign( design );
 
-											// Update fonts to the design defaults
-											setFonts( design.fonts );
+								// Update fonts to the design defaults
+								setFonts( design.fonts );
 
-											if ( isEnabled( 'gutenboarding/style-preview' ) ) {
-												push( makePath( Step.Style ) );
-											}
-										} }
-									>
-										<Spring
-											native
-											key={ design.slug }
-											from={ SHADOW_OFF }
-											to={ isFocused ? SHADOW_ON : SHADOW_OFF }
-										>
-											{ ( props2: React.CSSProperties ) => (
-												<animated.span style={ props2 } className="design-selector__image-frame">
-													<img alt={ design.title } src={ getDesignUrl( design ) } />
-												</animated.span>
-											) }
-										</Spring>
-										<span className="design-selector__option-overlay">
-											<span className="design-selector__option-name">{ design.title }</span>
-										</span>
-									</animated.button>
-								) }
-							</Spring>
-						);
-					} ) }
+								if ( isEnabled( 'gutenboarding/style-preview' ) ) {
+									push( makePath( Step.Style ) );
+								}
+							} }
+						>
+							<span className="design-selector__image-frame">
+								<img alt={ design.title } src={ getDesignUrl( design ) } />
```

Comment:
With `alt` and title again under the `img` this would prolly read to screen readers as "Design title. Design title.". Material for another PR however. :-)